src/twisted/test/test_factories.py

```diff
@@ -137,3 +137,17 @@ def test_parametrizedClock(self):
 
         factory.clientConnectionLost(FakeConnector(), None)
         self.assertEqual(len(clock.calls), 1)
+
+    def test_initialDelay(self):
+        """
+        Test that the first delay for reconnection is equal to the ``initialDelay``
+        attribute.
+        """
+        clock = Clock()
+        factory = ReconnectingClientFactory()
+        factory.jitter = None
+        factory.clock = clock
```

Comment:
I think that an explicit value can help with reading this test.
Also, removing the jitter should make it easier to check the result


```suggestion
        factory.clock = clock
        factory.initialDelay = 3
```

Reply:
> I think that an explicit value can help with reading this test.

If that's what you prefer, sure. Due to #5103, we'd need to set both `initialDelay` and `delay` though. Or actually, we'd only need to set `delay`, but that is even more confusing to the reader.

> Also, removing the jitter should make it easier to check the result

Since I already do that, do you mean that I need to add a comment explaining why:

```python
        # Remove jitter to make it easier to check the result
        factory.jitter = None
```
?

Reply:
Actually, #5103 is about subclassing, not about changing instance variables. They're related, but not the same.